policy/README.md

```diff
@@ -231,6 +240,7 @@ An individual `Rule` object is defined by the following fields:
 | `propulsion_types` | `propulsion_type[]`         | Optional   | Applicable vehicle [propulsion types][propulsion-types], default "all". |
 | `minimum`          | integer                     | Optional   | Minimum value, if applicable (default 0) |
 | `maximum`          | integer                     | Optional   | Maximum value, if applicable (default unlimited) |
+| `modality`         | [Modality][modalities]      | Optional   | Modality this rule should apply to (default `micromobility`) |
```

Comment:
You can specify a _policy_ that spans multiple modes, but you cannot specify a _rule_ that spans multiple modes. The reason being, each mode should have a different state machine, and because the rule also specifies states/events (and the sentiment can be slightly different between modes), you could get some really weird behavior.

Reply:
Okay, got it. I didn't think that through fully, but that makes sense to me.